grid-packages/ag-grid-docs/documentation/src/pages/license-pricing.tsx

```diff
@@ -9,104 +13,149 @@ const LicensePricing = () => {
-                                <div>
-                                    <p>
-                                        AG Grid Enterprise is a commercial product distributed under our{' '}
-                                        <a href="/eula/AG-Grid-Enterprise-License-Latest.html">EULA</a> and supported by
-                                        our technical staff. It has advanced functionality like Row Grouping, Range
-                                        Selection, Master / Detail, Server Side Row Model and{' '}
-                                        <a href="/javascript-data-grid/licensing/">more</a>. AG Grid Enterprise also comes
-                                        with <a href="/javascript-data-grid/integrated-charts/">Integrated Charts</a>,
-                                        allowing users to create charts using the grid's UI.
-                                    </p>
-                                    <p>
-                                        To evaluate AG Grid Enterprise you don&apos;t need our permission – all features
-                                        are unlocked. To temporarily hide the watermark and browser console errors
-                                        e-mail <a href="mailto:info@ag-grid.com">info@ag-grid.com</a> to get a temporary
-                                        evaluation key.
-                                    </p>
-                                    <p>
-                                        Once you&apos;re ready to begin development, please purchase an appropriate
-                                        license key from the options below.
-                                    </p>
-                                    <p>
-                                        Expanded definitions and FAQ responses are available further down the page. You
-                                        can e-mail us at any time on{' '}
-                                        <a href="mailto:info@ag-grid.com">info@ag-grid.com</a>.
-                                    </p>
-                                </div>
-
-                                <iframe
-                                    src="https://www.youtube.com/embed/20SLdu4wLtI"
-                                    title="YouTube video player"
-                                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
-                                    allowFullScreen
-                                ></iframe>
-                            </div>
-                        </div>
+
+                    <div className={styles.enterprise}>
+                        <h3>
+                            AG Grid Enterprise <EnterpriseIcon />
+                        </h3>
+                        <p>
+                            <b>AG Grid Enterprise</b> offers advanced functionality like{' '}
+                            <a href="/javascript-data-grid/grouping/">Row Grouping</a>,{' '}
```

Comment:
We'll need to import the `hostPrefix` from `utils/consts.js` for all these links eg,

```suggestion
                            <a href={`${hostPrefix}/javascript-data-grid/grouping/`}>Row Grouping</a>,{' '}
```